web/public/locales/es.json

```
@@ -156,15 +156,17 @@
   "misc": {
     "maintitle": "Emisiones de CO₂ del consumo eléctrico en tiempo real",
     "oops": "¡Ups! Estamos teniendo problemas con el servidor. Lo intentaremos de nuevo en unos segundos.",
-    "newversion": "¡Hay disponible una nueva versión! Pulsa <a onClick=\"location.reload(true);\">aquí</a> para recargar.",
+    "newversion": "¡Hay disponible una nueva versión!",
     "webgl-not-supported": "No se puede mostrar el mapa porque este navegador no es compatible con WebGL.",
     "retrynow": "Vuelve a intentarlo",
     "database-ad": "¿Buscas datos históricos? <a href=\"https://electricitymaps.com?utm_source=app.electricitymaps.com&utm_medium=referral\" target=\"_blank\">Echa un vistazo a nuestra Base de Datos!</a>",
     "api-ad": "¿Quieres datos en tiempo real en tu app o dispositivo? <a href=\"https://electricitymaps.com?utm_source=app.electricitymaps.com&utm_medium=referral\" target=\"_blank\">Prueba nuestra API!</a>",
     "legend": "Leyenda",
     "faq": "Preguntas frecuentes",
     "privacyPolicy": "Política de privacidad",
-    "legalNotice": "Aviso legal"
+    "legalNotice": "Aviso legal",
+    "dismiss": "Despedir",
```

Hi, this is better in this context:
```suggestion
    "dismiss": "Descartar",
```